### PR TITLE
New YiCAT updates

### PR DESCRIPTION
**Pull request by Cilla-luodan**

### CODE REVIEW

File: en-US/dita/RTC-NG/API/callback_irtcengineeventhandler_onremoteaudiotransportstats.dita [hunk 1]
```diff
@@ -28,59 +28,64 @@
...
             <dl outputclass="deprecated">
             <dlentry>
                 <dt>Deprecated:</dt>
-                <dd>Please use <xref keyref="onRemoteAudioStats"/> instead.</dd>
+                <dd>Please use <xref keyref="onRemoteAudioStats" /> instead.</dd>
             </dlentry>
             </dl>
             </p>
-            <p id="desc">This callback reports the transport-layer statistics, such as the packet loss rate and network time delay, once every two seconds after the local user receives an audio packet from a remote user. During a call, when the user receives the video packet sent by the remote user/host, the callback is triggered every 2 seconds.</p>
+            <p id="desc">This callback reports the transport-layer statistics, such as the packet loss rate and network time delay after the local user receives an audio packet from a remote user. During a call, when the user receives the audio packet sent by the remote user or host, the callback is triggered every 2 seconds.</p>
```
Comment (Chuchun1991) [inline en-US/dita/RTC-NG/API/callback_irtcengineeventhandler_onremoteaudiotransportstats.dita:59]: user包括host和audience，不能用or

File: en-US/dita/RTC-NG/API/callback_irtcengineeventhandler_onremoteaudiotransportstats.dita [hunk 2]
```diff
@@ -28,59 +28,64 @@
...
+                <pd>The ID of the remote user sending the audio streams.</pd>
             </plentry>
             <plentry>
                 <pt>delay</pt>
                 <pd>The network delay (ms) from the sender to the receiver.</pd>
             </plentry>
             <plentry>
                 <pt>lost</pt>
-                <pd>The packet loss rate (%) of the audio packet sent from the remote user.</pd>
+                <pd>The packet loss rate (%) of the audio packet sent from the sender to the receiver.</pd>
```
Comment (Chuchun1991) [inline en-US/dita/RTC-NG/API/callback_irtcengineeventhandler_onremoteaudiotransportstats.dita:83]: 跟前面的描述不一致，前面是remote user，这里是sender

File: en-US/dita/RTC-NG/API/class_videorenderingtracinginfo.dita [hunk 3]
```diff
@@ -69,7 +69,11 @@
...
                     <li>If the local user calls <apiname keyref="startMediaRenderingTracing"/> after successfully joining the channel, the value is the time interval from calling <apiname keyref="startMediaRenderingTracing"/> to when the remote user joins the channel. The unit is milliseconds.</li>
                     </ul>
-                    <note type="note">If the local user calls <apiname keyref="startMediaRenderingTracing"/> after the remote user joins the channel, the value is 0 and meaningless.</note>
+                    <note type="note">
+                        <ul>
+                            <li>If the local user calls <apiname keyref="startMediaRenderingTracing"/> after the remote user joins the channel, the value is 0 and meaningless.</li>
+                            <li>In order to improve the first frame speed of remote users, Agora recommends that the local user joins the channel when the remote user is in the channel to reduce this value.</li>
```
Comment (Suri539) [inline en-US/dita/RTC-NG/API/class_videorenderingtracinginfo.dita:75]: 建议改为：In order to reduce the time of rendering the first frame for remote users, Agora recommends that the local users join the channel after the remote user to reduce this value.
如果前面用speed的话，后面用 reduce，那你就是reduce the speed，意思变成了减慢xxx的速度，会有歧义。

File: en-US/dita/RTC-NG/API/class_videorenderingtracinginfo.dita [hunk 4]
```diff
@@ -82,7 +86,7 @@
                     <note type="note">
                     <ul>
                     <li>If the local user calls <apiname keyref="startMediaRenderingTracing"/> after setting the remote view, the value is 0 and meaningless.</li>
-                    <li>Agora recommends that you set the remote view before the remote user joins the channel, or set the remote view immediately after the remote user joins the channel, in order to reduce the value of this parameter and improve the speed of the first video frame rendering.</li>
+                    <li>In order to improve the first frame speed of remote users, Agora recommends that the local user sets the remote view before the remote user joins the channel, or sets the remote view immediately after the remote user joins the channel to reduce this value.</li>
```
Comment (Suri539) [inline en-US/dita/RTC-NG/API/class_videorenderingtracinginfo.dita:89]: Agora recommends that the local user sets the remote view before or immediately after the remote user joins the channel to reduce this value.

File: en-US/dita/RTC-NG/API/class_videorenderingtracinginfo.dita [hunk 5]
```diff
@@ -92,7 +96,11 @@
...
                     <li>If the local user calls <apiname keyref="startMediaRenderingTracing"/> after the remote user joins the channel, this value is the time interval from calling <apiname keyref="startMediaRenderingTracing"/> to subscribing to the remote video stream. The unit is milliseconds.</li>
                     </ul>
-                    <note type="note">If you call <apiname keyref="startMediaRenderingTracing"/> after subscribing to the remote video stream, the value is 0 and meaningless.</note>
+                    <note type="note">
+                        <ul>
+                            <li>If the local user calls <apiname keyref="startMediaRenderingTracing"/> after subscribing to the remote video stream, the value is 0 and meaningless.</li>
+                            <li>In order to improve the first frame speed of remote users, Agora recommends that after the remote user joins the channel, the local user immediately subscribes to the remote video stream to reduce this value.</li>
```
Comment (Suri539) [inline en-US/dita/RTC-NG/API/class_videorenderingtracinginfo.dita:102]: In order to reduce the time of rendering the first frame for remote users,

File: en-US/dita/RTC-NG/API/class_videorenderingtracinginfo.dita [hunk 6]
```diff
@@ -102,7 +110,11 @@
...
                     <li>If the local user calls <apiname keyref="startMediaRenderingTracing"/> after the remote user joins the channel, this value is the time interval from calling <apiname keyref="startMediaRenderingTracing"/> to receiving the remote video stream. The unit is milliseconds.</li>
                     </ul>
-                    <note type="note">If calling <apiname keyref="startMediaRenderingTracing"/> after receiving the remote video stream, the value is 0 and meaningless.</note></pd>
+                    <note type="note">
+                        <ul>
+                            <li>If the local user calls <apiname keyref="startMediaRenderingTracing"/> after receiving the remote video stream, the value is 0 and meaningless.</li>
+                            <li>In order to improve the first frame speed of remote users, Agora recommends that the remote user publishes video streams immediately after joining the channel, and the local user immediately subscribes to remote video streams to reduce this value.</li>
```
Comment (Suri539) [inline en-US/dita/RTC-NG/API/class_videorenderingtracinginfo.dita:116]: 同上

**Pull request merged**